### DISCUSSION TITLE
pin PD3 is reserved for t5uid1

### DISCUSSION THREAD

**Discussion by Safandula86** [Desuuuu/DGUS-reloaded-Klipper#31] [General]
Hi Desuuuu, thank for Your code. 
I have a standard 5 plus ender, unfortunately I have a bug related to bl touch. I can't change the port for it because it won't work. What can I do ?
Do you have any idea ?
![Zrzut ekranu 2022-04-26 190557](https://user-images.githubusercontent.com/51249171/165354722-f0196151-e463-41d4-8aba-bdf3aeadfe52.jpg)

**Comment by ghost** [comment]
something tells me the firmware isn't configured properly, thats just me...
